modules/js/CardTrait.js

```diff
@@ -279,17 +279,10 @@ define(['dojo', 'dojo/_base/declare'], (dojo, declare) => {
 
       var methodName = 'onClickCard' + this._action.charAt(0).toUpperCase() + this._action.slice(1);
       if (this[methodName] !== undefined) {
-        const CARD_BEER = 9;
-        const currentPlayer = this.gamedatas.players[this.player_id];
-        if (ocard.type === CARD_BEER && currentPlayer.bullets === currentPlayer.hp && this._action === 'playCard') {
-          this.confirmationDialog(
-            _(
-              'You have maximum amount of life points. Drinking a beer would currently have no effect. Do you still want to drink it?',
-            ),
-            () => {
-              this[methodName](card);
-            },
-          );
+        if (this._action === 'playCard' && card.options.confirmationMsg) {
+          this.confirmationDialog(card.options.confirmationMsg, () => {
```

Comment:
you need to add an underscore to translate the msg : 
confirmationDialog(_(card.options.confirmationMsg), ...

Reply:
But it's already in clienttranslate() method at backend! Why do we need to translate it twice?